_po4a-tools/po4a-cache.sh

```diff
@@ -6,8 +6,8 @@
 if [[ "$CACHE_HIT" == 'true' ]] ; then
     sudo cp --force --recursive ~/po4a/* /
 else
-    sudo apt install -yq gettext libsgmls-perl libyaml-tiny-perl opensp
-    wget -O po4a.deb https://github.com/jamulussoftware/assets/raw/main/po4a/po4a_0.66.deb
+    sudo apt install -yq gettext libsgmls-perl libyaml-tiny-perl opensp libsyntax-keyword-try-perl
+    wget -O po4a.deb https://github.com/jamulussoftware/assets/raw/main/po4a/po4a_0.68.deb
```

Comment:
Could you please upload the deb?

Reply:
It's in a PR in the assets repo. As I said in the description, I think #870 should be merged first, then the PR in assets, then this one. If I upload it now, the workflows won't find it.